packages/pwa-buildpack/lib/WebpackTools/plugins/UpwardDevServerPlugin.js

```diff
@@ -24,7 +24,6 @@ class UpwardDevServerPlugin {
             if (oldAfter) oldAfter(app, ...rest);
             // compression and security middlewares run before upward-js
             // but can run after everything else
```

Comment:
```suggestion
```

Also: Can you mention in the PR description that you decided to remove the `bestPractices` module from `upward-js` and that you tested the dev server to make sure it wasn't slower with this change?